README.md

```diff
@@ -1,2 +1,154 @@
-# hello-rails-react
-First rails-react project
+# README
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#test)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
```

Comment:
![image](https://github.com/maov19/hello-rails-react/assets/93039359/552803f5-2dc6-4961-8cf8-ba4edf79edbe)

- [ ] It would be nice if you fix the highlighted links in the image above to make your readme descriptive and professional 👍 Kindly fix them or remove them 😊 Links which are broken are the following ⬇️

```sh
1. Live Demo
2. FAQ
```

Reply:
Fixed! thanks for the observation 👀 
I just removed them.